### ISSUE TITLE
Error when trying to load Source File in waveform view

### ISSUE THREAD

**Issue by aron7awol** [3ll3d00d/beqdesigner#177]
I loaded a previously saved BEQ and modified the filters for it.  Popped up the waveform view, selected the signal, and tried to load Source File and received the following error:

![image](https://user-images.githubusercontent.com/42757020/50575768-4d23dd00-0dd2-11e9-82eb-5448deb49c49.png)

**Comment by 3ll3d00d** [comment]
can you confirm exactly what you mean by "loaded a previously saved BEQ"?

is it via "load project"? if so, can you share the beq file?

**Comment by aron7awol** [comment]
Yes, I loaded project.  It's an older saved project where the signal didn't have the source file path saved with it, but I thought in that case I could manually open the source file.

[FindingNemo.zip](https://github.com/3ll3d00d/beqdesigner/files/2719600/FindingNemo.zip)
